client_apis/python/src/tests/integration/watchlist_test.py

```diff
@@ -0,0 +1,420 @@
+            cb.watchlist_modify(
+                id=add_result["id"],
+                watchlist=modifications
+            )
+
+        self.assertEqual(err.exception.response.status_code, 500)
+        return
+
+    def test_get_actions_for_watchlist(self):
+        add_result = cb.watchlist_add(
+            type="modules",
+            name="%s %s" % (name_prefix, datetime.now().isoformat(' ')),
+            search_query="q=is_executable_image%3Afalse&cb.urlver=1&sort=server_added_timestamp%20desc",
+            readonly=False
+        )
+        watchlist_id = add_result["id"]
+        self.assertGreater(add_result["id"], 0)
+
+        actions = cb.watchlist_action_get(watchlist_id=watchlist_id)
+        self.assertIsNotNone(actions)
+        self.assertEqual(len(actions), 0)
+        return
+
+    def test_add_watchlist_action(self):
+        add_result = cb.watchlist_add(
+            type="modules",
+            name="%s %s" % (name_prefix, datetime.now().isoformat(' ')),
+            search_query="q=is_executable_image%3Afalse&cb.urlver=1&sort=server_added_timestamp%20desc",
+            readonly=False
+        )
+        watchlist_id = add_result["id"]
+        self.assertGreater(add_result["id"], 0)
+
+        actions = cb.watchlist_action_get(watchlist_id=watchlist_id)
+        self.assertIsNotNone(actions)
+        self.assertEqual(len(actions), 0)
+
+        add_action_result = cb.watchlist_action_add(
+            watchlist_id=watchlist_id,
+            action_type='email',
+            email_recipient_user_ids=[1]
+        )
+        self.assertGreater(add_action_result["action_id"], 0)
+
+        actions = cb.watchlist_action_get(watchlist_id=watchlist_id)
+        self.assertIsNotNone(actions)
+        self.assertEqual(len(actions), 1)
+        self.assertEqual(actions[0]["id"], add_action_result["action_id"])
+        return
+
+    def test_add_invalid_watchlist_action(self):
+        add_result = cb.watchlist_add(
+            type="modules",
+            name="%s %s" % (name_prefix, datetime.now().isoformat(' ')),
+            search_query="q=is_executable_image%3Afalse&cb.urlver=1&sort=server_added_timestamp%20desc",
+            readonly=False
+        )
+        watchlist_id = add_result["id"]
+        self.assertGreater(add_result["id"], 0)
+
+        actions = cb.watchlist_action_get(watchlist_id=watchlist_id)
+        self.assertIsNotNone(actions)
+        self.assertEqual(len(actions), 0)
+
+        with self.assertRaises(ValueError):
+            cb.watchlist_action_add(
+                watchlist_id=watchlist_id,
+                action_type='invalid_action',
+                email_recipient_user_ids=[1]
+            )
+
+        actions = cb.watchlist_action_get(watchlist_id=watchlist_id)
+        self.assertIsNotNone(actions)
+        self.assertEqual(len(actions), 0)
+        return
+
+    @unittest.skip("Duplicate action type validation has not been implemented")
```

Comment:
I don't think you actually want to have this skipped, right?  The test has exposed a bug, so should be left active, and the error will go away when the bug is fixed.